src/core/remote.js

```diff
@@ -1610,7 +1610,7 @@ Remote.prototype.requestBookOffers = function(options_, callback_) {
   const request = new Request(this, 'book_offers');
 
   request.message.taker_gets = {
-    currency: Currency.json_rewrite(gets.currency, {force_hex: true})
```

Comment:
I don't think so. Out atm tho in droid. Look for where iso_code gets set.
It's only set for simple currencies? Interest bearing (i know they are
deprecated) currencies don't set them too?